datastore/modeladapter_test.go

```diff
@@ -43,17 +42,6 @@ func TestModelNameEmpty(t *testing.T) {
 	}
 }
 
-func TestModelNameInvalidChar(t *testing.T) {
```

Comment:
This kind of changes deserve some tests

Reply:
This test is no longer valid... that's why I removed it.

Reply:
Sure, but I was pointing if another similar test would be needed like TestValidateCaseInsensitive()

Reply:
As discussed, this is covered in TestModelNameUpperChar